db/schema.rb

```diff
@@ -73,6 +73,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.boolean "successful", default: false
+    t.string "square_payment_id"
```

Comment:
Is there a way to verify that either square_payment_id != || stripe_id != null but not both?

Reply:
Yeah there is!